### DISCUSSION TITLE
Support PROXY protocol when running behind Load Balancer

### DISCUSSION THREAD

**Discussion by dorinclisu** [bluenviron/mediamtx#477] [Questions]
Is it difficult to support it? https://www.haproxy.org/download/1.8/doc/proxy-protocol.txt

The reason is, behind a load balancer all rtsp server request IP's are of the balancer itself, therefore losing the ability to whitelist client IP's.

The load balancer I'm using does have a proxy toggle, and specifically warns it that the application must support it if enabled. I did try it on and the request wouldn't reach the rtsp server anymore so I assume it doesn't support it yet.

**Comment by dorinclisu** [comment]
Any idea?

**Comment by aler9** [comment]
moved here #602
